src/main/java/net/snowflake/client/core/ResultUtil.java

```diff
@@ -26,7 +26,7 @@ public class ResultUtil {
 
   public static final int MILLIS_IN_ONE_DAY = 86400000;
   public static final int DEFAULT_SCALE_OF_SFTIME_FRACTION_SECONDS =
-      3; // default scale for sftime fraction seconds
+      3; // default scale for SFTime fraction seconds
```

Comment:
This was the only instance of this casing, and it seemed simpler to just make it match the type (unlike above where I split the term)